### PR TITLE
Add a new topic about translatomics

### PR DESCRIPTION
**Pull request by ldyang14**
Hello, everyone!

My partner and I plan to create a new topic about translatomics. We have written some drafts, and the rest will be improved and submitted one after another. Please give us some feedback to make further improvements. Thank you for your time.

Best wishes,
Ludong Yang

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,15 +0,0 @@
-source "https://rubygems.org"
-gem "nokogiri", ">= 1.10.4"
```
Comment (bgruening) [inline Gemfile]: please leave those files included

Comment (ldyang14) [reply Gemfile]: Thanks a lot for your feedback. I have added those files.

**Comment by ldyang14** [comment]
@bgruening Thank you very much for your valuable suggestions!

We will improve the remaining tutorials about translatomics as soon as possible and submit them together. Because we have constructed a Galaxy-based translatome workbench (https://db.cngb.org/galaxy/) and some workflows for ribosome profiling analysis. Therefore, we still hope to be able to create a new topic for that.
I sincerely listen to your further feedback. Thanks again!

**Comment by hexylena** [comment]
#1719 appears to be a superset of this pull request so I will close this one and reviews can occur on the newer PR. Thanks @ldyang14

**Pull request closed without merging**